src/main/java/forum/entity/Thread.java

```diff
@@ -0,0 +1,84 @@
+package forum.entity;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="thread")
+public class Thread {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "id")
```

Comment:
Hmm do you have to defined the `name` if it is the same as the property? https://docs.oracle.com/javaee/5/api/javax/persistence/Column.html

Reply:
Yeah it's not necessary but I figured maybe it's clearer that way. Is it best practice to leave the definition out?